### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by dhollinger**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
+
+## [v2.0.0](https://github.com/voxpupuli/puppet-rsyslog/tree/v2.0.0) (2017-11-27)
+[Full Changelog](https://github.com/voxpupuli/puppet-rsyslog/compare/0.1.0...v2.0.0)
```
Comment (alexjfisher) [inline CHANGELOG.md]: The last release wasn’t 0.1.0

File: HISTORY.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## 1.1.0
```
Comment (alexjfisher) [inline HISTORY.md]: Does this line also need a release date for the changelog generator to detect 1.1.0 as the last release?

Comment (dhollinger) [reply HISTORY.md]: I can't see why it does. Github treats tags as releases by default. Not sure why it sees 0.1.0, but not 0.2.0, 1.0.0, or 1.1.0

Comment (alexjfisher) [reply HISTORY.md]: I think gcg parses the HISTORY.md to work out the `since_tag`.

Comment (dhollinger) [reply HISTORY.md]: Do we need to add an Entry in HISTORY.md for 0.1.0?

Comment (alexjfisher) [reply HISTORY.md]: Not for gcg.  You only technically need to fix up the most recent entry.
`## 1.1.0 (2017-10-17)` should do it.

**Pull request merged**